MathTrainer/MathTrainer/TrainViewController.swift

```diff
@@ -113,4 +125,20 @@ final class TrainViewController: UIViewController {
             }
         }
     }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
```

Comment:
unwind segue must be implemented in the ViewController 
you must not have an access to ViewController from TrainViewController
i was explaining this in the life review, you can find it in the TG chat, if not contact to me in the TG.